scripts/create-parameters-file.sh

```diff
@@ -8,7 +8,7 @@ ORGNAME="BuilditAzureSandbox"
 PROJECTNAME="BuilditAzureRig"
 PROJECTID="7b202733-9515-4424-be96-5fb9e2ccc0f3"
 GITORG="buildit"
-GITREPO="digitalrig-metal-azure"
+GITREPO="slackbot"
```

Comment:
can we add a TODO or something that we should prompt for this rather than being "hardcoded" in the script file...also need to document this in the README

Reply:
I will add to backlog